### PR TITLE
Babel Team Meeting Notes 2017-05-31

### PR DESCRIPTION
**Pull request by nathanhammond**
Notes are up, review for correctness and completeness!

### CODE REVIEW

File: 2017-05/may-31.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- [X] (Nathan) Publish meeting notes.
+- [ ] (Henry) Specify the complete list of outstanding tasks on the plugin ordering PR.
+ 
+## Discussion
+ 
+### Weekly Meetings!
+ 
+- Introductions!
+- We have them now. Coming up with a time was hard. I'm sorry if we failed to accommodate your time zone.
+- Most people seem comfortable with our selected time.
```
Comment (hzoo) [inline 2017-05/may-31.md]: We should put the actual time here (Wed 9am EST, or multiple time zones)

File: 2017-05/may-31.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ 
+### Summer of Code!
+ 
+- We have four new people on the team!
```
Comment (hzoo) [inline 2017-05/may-31.md]: should we add the SoC names here?

Comment (xtuc) [inline 2017-05/may-31.md]: Yes I think so

File: 2017-05/may-31.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+|Sydney (Australia - New South Wales) | 11:00:00 pm | AEST | UTC+10 hours|
+|Corresponding UTC (GMT) |  13:00:00|
+ 
+### Summer of Code!
+ 
+- We have four new people on the team!
+  - [GSoC](https://twitter.com/gsoc): Peeyush Kushwaha, :octocat: [@peey](https://github.com/peey), :bird: [@peeyFTW](https://twitter.com/peeyFTW)
+  - [GSoC](https://twitter.com/gsoc): Karl Cheng, :octocat: [@Qantas94Heavy](https://github.com/Qantas94Heavy), :bird: [@qantas94heavy](https://twitter.com/qantas94heavy)
+  - [RGSoC](https://twitter.com/RailsGirlsSoC): Kara de la Marck, :octocat: [@MarckK](https://github.com/MarckK), :bird: [@KaraMarck](https://twitter.com/KaraMarck)
+  - [RGSoC](https://twitter.com/RailsGirlsSoC): Emma Sofia Pohjalainen :octocat: [@sofiapoh](https://github.com/sofiapoh), :bird: [@sofiapoh](https://twitter.com/sofiapoh)
```
Comment (nathanhammond) [inline 2017-05/may-31.md]: Sofia isn't going to be working on this, it's going to be Emma Deacon.

**Pull request merged**